<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>683</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelDate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelDuration">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDuration">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTitle">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>671</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Baidu Speaker Barcode Check Test</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSubTitle">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);
</string>
     </property>
     <property name="text">
      <string>Manufacturing Test System</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVersion">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255);</string>
     </property>
     <property name="text">
      <string>Version: 2019.11.19.V1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLogo">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rc.qrc">:/res/images/MINAMI_logo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>1161</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
